BatMobile.java

```diff
@@ -0,0 +1,14 @@
+public class BatMobile extends Car {
+    public void shoot(){
+        System.out.println("The BatMobile is shooting!");
+    }
+    public void bulletCapacity(int howMuch){

```

Comment:
Нарушение Code Convention:
```suggestion
    public void bulletCapacity(int howMuch) {
```